From dd8624a5fba19840e585f50a2326235446f98cb0 Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Sat, 23 Feb 2019 00:22:00 +0200
Subject: [PATCH 8/9] legacy: Resolve hal_camera_default denials

* As seen on msm8994 and msm8916 when camera provider is
  running in hwbinder mode

Change-Id: I39dfa0c55401480c6bb7dbeb55b6f3efa3edc96c
---
 legacy-common/hal_camera_default.te | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/legacy-common/hal_camera_default.te b/legacy-common/hal_camera_default.te
index 670d8bc..a51f70d 100644
--- a/legacy-common/hal_camera_default.te
+++ b/legacy-common/hal_camera_default.te
@@ -1 +1,4 @@
+allow hal_camera_default camera_data_file:dir search;
+allow hal_camera_default camera_data_file:sock_file write;
+
 allow hal_camera_default mpctl_socket:dir search;
-- 
2.17.1

